sdk/spring/spring-cloud-azure-integration-tests/src/test/resources/application-appconfiguration.yml

```diff
@@ -1,5 +1,7 @@
 spring:
   cloud:
     azure:
+      credential:
+        token-credential-bean-name: integrationTestTokenCredential
```

Comment:
Could you please write a doc to explain how the token credential is selected for kinds of conditions?

Reply:
Already recorded in backlog, will be added in the next Sprint